### DISCUSSION TITLE
Meaning of offline model

### DISCUSSION THREAD

**Discussion by thewh1teagle** [k2-fsa/sherpa-onnx#1372] [Q&A]
What's the meaning of offline / online model and in general, the terms in sherpa-onnx?
I assume it's not internet-online right?

**Comment by csukuangfj** [comment]
Please see the FAQ  in the doc

You.can also look at Wikipedia for what are online algorithms and offline algorithms.

**Reply by thewh1teagle** [reply]
I see. Online algorithms are nicer because we don't need the whole input at once to start ingest it

**Reply by csukuangfj** [reply]
by the way, in some.cases you.have all the input at hand.

also, offline models usually have a higher accuracy and they also run faster.
